var/spack/repos/builtin/packages/r-ks/package.py

```diff
@@ -0,0 +1,42 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class RKs(RPackage):
+    """Kernel smoothers for univariate and multivariate data."""
+
+    homepage = "https://cran.r-project.org/package=ks"
+    url      = "https://cran.r-project.org/src/contrib/Archive/ks/ks_1.11.2.tar.gz"
+
+    version('1.11.2', sha256='9dfd485096e1e67abc7dfcb7b76a83de110dd15bcfeffe5c899605b3a5592961')
+
+    depends_on('r@2.10:')
+
+    depends_on('r-fnn@1.1:')
+    depends_on('r-kernlab')
+    depends_on('r-mclust', type=('build', 'run'))
+    depends_on('r-multicool')
+    depends_on('r-mvtnorm@1.0:', type=('build', 'run'))
```

Comment:
Missing a few dependencies:

`r-kernsmooth@2.22:`
`r-matrix`
`r-mgcv`

Also, all of these should be `type=('build', 'run')`, including `r`.